dotnet.cmd

```diff
@@ -16,7 +16,7 @@ set Platform=
 :: Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
 set DOTNET_MULTILEVEL_LOOKUP=0
```

Comment:
`DOTNET_MULTILEVEL_LOOKUP` does not do anything for multiple releases now. I think it can be deleted everywhere.